resources/views/Listings/show.blade.php

```diff
@@ -14,16 +14,17 @@
     <div class="right">
         <div class="d-flex">
             <h1>{{$listing->product->product_name}}</h1>
-            {{-- CHECK OF PRODUCT AL FAVORIET IS OF NIET (IF ELSE) --}}
-            {{-- geen favoriet --}}
-            <form class="d-flex" action="">
-                @csrf
-                <button class="icon-button"><i style="font-size: 32px" class="bi bi-heart"></i></button>
-            </form>
-            {{-- al wel favoriet --}}
-            <form class="d-flex" action="">
-                <button class="icon-button"><i style="font-size: 32px" class="bi bi-heart-fill"></i></button>
-            </form>
+                @if(!$favorite->isEmpty())
```

Comment:
Doe ook een check of hij uberhaupt bestaat. of lost dit anders op. Als je inet ingelogd bent en nu een advertentie wilt zijn krijg je een error.